tests/integration/security/authorization_test.go

```diff
@@ -435,6 +437,81 @@ func TestAuthorization_Deny(t *testing.T) {
 		})
 }
 
+// TestAuthorization_NegativeOperation tests the authorization policy with negative match.
+func TestAuthorization_NegativeOperation(t *testing.T) {
```

Comment:
There is already a `TestAuthorization_NegativeMatch()` for negative matching fields, could you update it instead of adding a new one?